main/src/main/scala/sbt/Keys.scala

```diff
@@ -78,174 +85,511 @@ import org.apache.logging.log4j.core.Appender
 import sbt.BuildSyntax._
 
 object Keys {
-  val TraceValues = "-1 to disable, 0 for up to the first sbt frame, or a positive number to set the maximum number of frames shown."
+  val TraceValues =
+    "-1 to disable, 0 for up to the first sbt frame, or a positive number to set the maximum number of frames shown."
 
   // logging
-  val logLevel = SettingKey[Level.Value]("log-level", "The amount of logging sent to the screen.", ASetting)
-  val persistLogLevel = SettingKey[Level.Value]("persist-log-level", "The amount of logging sent to a file for persistence.", CSetting)
-  val traceLevel = SettingKey[Int]("trace-level", "The amount of a stack trace displayed.  " + TraceValues, ASetting)
-  val persistTraceLevel = SettingKey[Int]("persist-trace-level", "The amount of stack trace persisted.", CSetting)
-  val showSuccess = SettingKey[Boolean]("show-success", "If true, displays a success message after running a command successfully.", CSetting)
-  val showTiming = SettingKey[Boolean]("show-timing", "If true, the command success message includes the completion time.", CSetting)
-  val timingFormat = SettingKey[java.text.DateFormat]("timing-format", "The format used for displaying the completion time.", CSetting)
-  val extraLoggers = SettingKey[ScopedKey[_] => Seq[Appender]]("extra-loggers", "A function that provides additional loggers for a given setting.", DSetting)
-  val logManager = SettingKey[LogManager]("log-manager", "The log manager, which creates Loggers for different contexts.", DSetting)
-  val logBuffered = SettingKey[Boolean]("log-buffered", "True if logging should be buffered until work completes.", CSetting)
-  val sLog = SettingKey[Logger]("setting-logger", "Logger usable by settings during project loading.", CSetting)
+  val logLevel =
+    SettingKey[Level.Value]("log-level", "The amount of logging sent to the screen.", ASetting)
+  val persistLogLevel =
+    SettingKey[Level.Value]("persist-log-level",
+                            "The amount of logging sent to a file for persistence.",
```

Comment:
uh... do we want to change this like this? I know the former is longer than 100 columns, but I thinks it's about 1000x more readable. We should exclude these blocks, maybe the whole file.